### PR TITLE
Tackling user feedback

### PR DESCRIPTION
**Pull request by lidiazuin**

### CODE REVIEW

File: modules/ROOT/pages/type-definitions/directives/indexes-and-constraints.adoc [hunk 1]
```diff
@@ -104,7 +104,7 @@ When you run xref::/type-definitions/directives/indexes-and-constraints.adoc#_as
 
 [source, cypher, indent=0]
 ----
-CALL db.index.fulltext.createNodeIndex("ProductName", ["Product"], ["name"])
+CREATE FULLTEXT INDEX ProductName FOR (n:Product|Book) ON EACH [n.Product, n.name]
```
Comment (darrellwarde) [inline modules/ROOT/pages/type-definitions/directives/indexes-and-constraints.adoc]: Where has the label `Book` come from here?

File: modules/ROOT/pages/ogm/reference.adoc [hunk 2]
```diff
@@ -428,7 +428,7 @@ Returns a `Promise` that resolves to the equivalent of the Mutation response for
 
 For the User with name "John", update their name to be "Jane":
 
-[source, javascript, indent=0]
+[source, graphql, indent=0]
```
Comment (darrellwarde) [inline modules/ROOT/pages/ogm/reference.adoc]: `javascript` was correct here!

**Comment by neo-technology-commit-status-publisher** [comment]
Thanks for the documentation updates.

The preview documentation has now been torn down - reopening this PR will republish it.

**Review by darrellwarde — COMMENTED** [review]
Some specific comments below, but I'm generally confused by this PR - the code examples in the table don't particularly match to the row, and I'm not 100% clear on what they are trying to demonstrate. Could you shed some light on the feedback at hand that led to this @lidiazuin?

File: modules/ROOT/pages/type-definitions/directives/indexes-and-constraints.adoc (hunk 1 above)
Comment (lidiazuin) [reply modules/ROOT/pages/type-definitions/directives/indexes-and-constraints.adoc]: Nowhere, I was just trying to adapt the example I got from ChatGPT 😅 . Would the correct option be like this?

```suggestion
CREATE FULLTEXT INDEX ProductName FOR (n:ProductName) ON EACH [n.Product, n.name]
```

Comment (darrellwarde) [reply modules/ROOT/pages/type-definitions/directives/indexes-and-constraints.adoc]: It should be:

```suggestion
CREATE FULLTEXT INDEX ProductName FOR (n:Product) ON EACH [n.name]
```

🙂

**Comment by darrellwarde** [comment]
@lidiazuin, I think merging these changes to the OGM reference will render the page quite inconsistent. If you go to the preview, you'll see that the table of argument for update now has three columns, but the other four operations still just have a two column table. We should probably be considering making it consistent across the board.

**Comment by lidiazuin** [comment]
> @lidiazuin, I think merging these changes to the OGM reference will render the page quite inconsistent. If you go to the preview, you'll see that the table of argument for update now has three columns, but the other four operations still just have a two column table. We should probably be considering making it consistent across the board.

I agree. But then would you rather add examples to all cases or keep them the way they are?

**Comment by darrellwarde** [comment]
That's a lot of examples to add. 😅 In the long term, I would like to get rid of this API reference page entirely, and switch to more tutorial/how-to style. With that in mind, would it be worth the effort now?

**Comment by lidiazuin** [comment]
Not really, no. So I will revert that part!

File: modules/ROOT/pages/ogm/reference.adoc [hunk 3]
```diff
@@ -37,7 +37,7 @@ Accepts a single argument `name` of type `string`.
 
 For the following type definitions:
 
-[source, graphql, indent=0]
+[source, javascript, indent=0]
```
Comment (darrellwarde) [inline modules/ROOT/pages/ogm/reference.adoc]:
```suggestion
[source, graphql, indent=0]
```

**Review by darrellwarde — APPROVED** [review]
Looks good, just the one suggestion that needs updating!

**Pull request merged**